hts-ft/script-hts-ft.js

```diff
@@ -58,28 +54,18 @@ async function scriptHtsFungibleToken() {
         .setInitialSupply(1_000_000)
         // Configure token access permissions: treasury account, admin, freezing
         .setTreasuryAccountId(operatorId)
-        // Set the admin key of the token
-        .setAdminKey(adminPublicKey) //ToDo: replace with adminKey
         // Set the freeze default value to false
         .setFreezeDefault(false)
-        // Freeze the transaction to prepare for signing
-        .freezeWith(client);
+        // Sign the transaction with the operator account key and submit to the network
+        .execute(client);
 
     // Get the transaction ID of the transaction. The SDK automatically generates and assigns a transaction ID when the transaction is created
     const tokenCreateTxId = tokenCreateTx.transactionId;
     logger.log('The token create transaction ID: ',
         tokenCreateTxId.toString());
 
-    // Sign the transaction with the account key that will be paying for this transaction
-    const tokenCreateTxSigned = await tokenCreateTx.sign(adminPrivateKey);
-
-    //ToDo: Sign with the admin key
-
-    // Submit the transaction to the Hedera Testnet
-    const tokenCreateTxSubmitted = await tokenCreateTxSigned.execute(client);
-
     // Get the transaction receipt
-    const tokenCreateTxReceipt = await tokenCreateTxSubmitted.getReceipt(client);
```

Comment:
Hmm... 🤔

Reply:
updated PR to add these steps back thanks!